test/Microsoft.AspNet.Mvc.FunctionalTests/Logging/LoggingTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if ASPNET50
```

Comment:
This test is failing in CoreCLR due to Json.net serialization failures...Will investigate more...